pkg/providers/tinkerbell/stack/stack.go

```diff
@@ -234,15 +234,25 @@ func getURIDir(uri string) (string, error) {
 	return uri[:index], nil
 }
 
-// CheckLocalBootsExistence determines whether Boots is already running locally on a Docker container
-// Returns nil if the container is not found and an error otherwise
-func (s *Installer) CheckLocalBootsExistence(ctx context.Context) error {
+// CleanupLocalBoots determines whether Boots is already running locally
+// and either cleans it up or errors out depending on the `remove` flag
+func (s *Installer) CleanupLocalBoots(ctx context.Context, remove bool) error {
```

Comment:
This is okay for now. Just a nit, 'CleanupLocalBoots' should just do clean up. Coming inside a function called 'CleanupLocalBoots' and then cleaning up only if 'remove'(force) is set -- could be changed to read better. Thanks for taking care of this!